android/modules/filesystem/src/java/ti/modules/titanium/filesystem/FilesystemModule.java

```diff
@@ -81,6 +81,10 @@ public boolean isExternalStoragePresent()
 	@Kroll.method
 	public FileProxy getFile(KrollInvocation invocation, Object[] parts)
 	{
+		//If directory doesn't exist, return
+		if (parts[0] == null) {
```

Comment:
If the file name is null, it won't make it to Java. I think the check should be efficient